.gitignore

```diff
@@ -59,6 +59,7 @@ _ReSharper*
 
 # Others
 
+SolutionInfo.cs
```

Comment:
If `SolutionInfo.cs` is already a tracked file, adding it to the `.gitignore` won't stop it from being tracked.  The `.gitignore` only affects new files.  Is this ignoring a copy of `SolutionInfo.cs` that isn't committed in the repo?

Reply:
The solutioninfo.cs change was recommended by Oliver that I add it because one of the local build scripts was modifying the the shared file solutioninfo.cs file (to update the version) and then was getting picked as a change.  After adding this to the .gitignore file my Visual Studio is now no longer showing changes to the file, across the projects, and has the icon to be ignored.  Is there another way I should be doing this? Should this be removed?

Reply:
Unless I'm misunderstanding the scenario, adding the file to the `.gitignore` did not cause changes to the file to be ignored (try manually making a change to the file and see if it shows as a change).

Reply:
I just made a change on my end and it seems to be  being ignored. See attached pic.

![image](https://user-images.githubusercontent.com/39993943/58034257-88fadd00-7afc-11e9-83bf-d058bd94b430.png)

Reply:
If there is confusion I can just remove this change from this PR and we can discuss the scenario/issue at tomorrows meeting and see if there is a better way forward. @ohine has a better idea of the issue and can probably explain better than I. I'm not sure why others haven't ran into the issue on their local env. - it had to do with the version on the DLL's being incorrect and not loading.

Reply:
Weird, I still see a change if I edit that file, in VS 2017 and on the command line.  Thanks for removing, we can look at what might need to change around that separately.  
![SolutionInfo.cs listed under Changes in Visual Studio 2017's Team Explorer](https://user-images.githubusercontent.com/59507/58035482-7b3e5b00-7aee-11e9-94de-04daf69e8b2c.png)